cmake/Config.cmake.in

```diff
@@ -0,0 +1,15 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+find_dependency(crypto3 REQUIRED)
```

Comment:
Boost is also a direct dependency, so probably should add it here

Reply:
Thanks, added